Cargo.toml

```diff
@@ -48,6 +48,7 @@ toml = { version = "0.8", optional = true }
 
 [dev-dependencies]
 pretty_assertions = "1.2.1"
+derive_more = { version = "2.0.1", features = ["debug"] }
```

Comment:
I unfortunately had to add a new dependency just for the test because I couldn't think of any derive macros in the standard library that use attributes on struct fields.

Reply:
I can't think of any `std` macros doing that either, so yeah, totally fine. After all, users don't care about dev dependencies.